app/src/test/scala/org/alephium/explorer/api/model/PaginationSpec.scala

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 The Alephium Authors
+// This file is part of the alephium project.
+//
+// The library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the library. If not, see <http://www.gnu.org/licenses/>.
+
+package org.alephium.explorer.api.model
+
+import org.alephium.explorer.AlephiumFutureSpec
+import org.alephium.explorer.Generators._
+import org.alephium.explorer.persistence.{DatabaseFixtureForEach, DBRunner}
+
+class PaginationSpec extends AlephiumFutureSpec with DatabaseFixtureForEach with DBRunner {
+
+  "pagination" should {
```

Comment:
I would maybe precise here: `paginationGen` because we are testing the generator here, not the actual `Pagination`

Reply:
Oh you are right. I will just remove the test yea? Doesn't make sense to be testing a generator :)

Reply:
yeah let's remove it